bagitobjecttransfer/manage.py

```diff
@@ -38,11 +38,26 @@ def initialize_debugger():
             debugpy.listen(('0.0.0.0', port))
 
 
+def check_yuglify_binary():
+    from django.conf import settings
+    from django.core.exceptions import ImproperlyConfigured
+
+    if not os.path.exists(settings.YUGLIFY_BINARY):
```

Comment:
Yes, take a look at `bagitobjecttransfer.settings.base`. If someone wanted to define it manually they can, otherwise it tries to select the correct binary based on on your platform (which might not be working, see my other comment).

Reply:
Ah this makes sense.